docs/docs/data-sources/dynamodb.md

```diff
@@ -0,0 +1,37 @@
+---
+sidebar_position: 11
+---
+
+# DynamoDB
+
+ToolJet can connect to DynamoDB to read and write data.
+
+## Connection
+
+Please make sure the host/ip of the database is accessible from your VPC if you have self-hosted ToolJet. If you are using ToolJet cloud, please whitelist our IP.
+
+To add a new DynamoDB, click on the `+` button on data sources panel at the left-bottom corner of the app editor. Select DynamoDB from the modal that pops up.
+
+ToolJet requires the following to connect to your DynamoDB.
+
+- **Region**
+- **Access key**
+- **Secret key**
+
+It is recommended to create a new database user so that you can control the access levels of ToolJet.
```

Comment:
This might not be relevant in the case of DynamoDB. Please check how the access roles are set for DynamoDB.

Reply:
Hi @Navaneeth-pk 
I have a very basic understanding of DynamoDB. Please help with the details

Reply:
"It is recommended to create a new IAM user for the database so that you can control the access levels of ToolJet."
Will this work?
@Navaneeth-pk

Reply:
yes works 👍